elastic/kibana
# [Fleet] Allow to enable global logs and metrics dataview creation with config enableManagedLogsAndMetricsDataviews

## Description 

Resolve #202807

Introduce a new `xpack.fleet.enableManagedLogsAndMetricsDataviews` to allow to disable the creation of the managed `logs-*` and `metrics-*` dataview.

## How to test

Run fleet `xpack.fleet.enableManagedLogsAndMetricsDataviews: false` and verify `logs-*` and `metrics-*` are not created


## Todo

- [x] Document the config flag
